packages/glsp-sprotty/src/api-override.ts

```diff
@@ -198,3 +199,53 @@ export interface IActionDispatcher extends SIActionDispatcher {
 }
 
 export type IActionDispatcherProvider = () => Promise<IActionDispatcher>;
+
+/**
+ * Data structure that is passed to the `onCommandExecuted` event.
+ */
+export interface CommandExecutionData {
+    /** The command that has been executed successfully */
+    command: ICommand;
+    /** The new model root after the command execution */
+    newRoot: GModelRoot;
+}
+
+export interface ICommandStack extends SICommandStack {
+    /**
+     * Executes the given command on the current model and returns a
+     * Promise for the new result.
+     *
+     * Unless it is a special command, it is pushed to the undo stack
+     * such that it can be rolled back later and the redo stack is
+     * cleared.
+     */
+    execute(command: ICommand): Promise<GModelRoot>;
+
+    /**
+     * Executes all of the given commands. As opposed to calling
+     * execute() multiple times, the Viewer is only updated once after
+     * the last command has been executed.
+     */
+    executeAll(commands: ICommand[]): Promise<GModelRoot>;
```

Comment:
Any reason you re-specified this method? Seems like we could just take it from the super interface.